client/components/Tracker/Form/Form.jsx

```diff
@@ -37,7 +37,7 @@ const Form = ({ getList, renderCount, user }) => {
           <ThoughtsContainer>
 
             <h2>Food For Thought</h2>
-            <ThoughtInput type="text" onChange={(event) => updateThoughts(event.target.value)} />
+            <ThoughtInput type="textarea" onChange={(event) => updateThoughts(event.target.value)} />
```

Comment:
I don't think `type="textarea"` works here. I think you can change the `ThoughtInput` to be a textarea though with `styled.textarea` but you'd need to adjust the height / width again. I verified that it's still an input by inspecting with devtools.